emukit/bayesian_optimization/acquisitions/max_value_entropy_search.py

```diff
@@ -0,0 +1,119 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+
+from typing import Union, Callable
+
+import scipy
+import numpy as np
+from scipy.stats import norm
+from scipy.optimize import bisect
+
+from ...core.acquisition import Acquisition
+from ...core.interfaces import IModel
+from ...core.parameter_space import ParameterSpace
+
+from ..interfaces import IEntropySearchModel
+from ...experimental_design import RandomDesign
+
+
+
+class MaxValueEntropySearch(Acquisition):
+
+    def __init__(self, model: Union[IModel, IEntropySearchModel], space: ParameterSpace,
+                 num_samples: int = 10, grid_size: int = 5000) -> None:
+
+        """
+        MES acquisition function approximates the distribution of the value at the global
+        minimum and tries to decrease its entropy. See this paper for more details:
+        Z. Wang, S. Jegelka
+        Max-value Entropy Search for Efficient Bayesian Optimization
+        ICML 2017
+
+        This part of the code is heavily based on 
```

Comment:
What do you mean by "heavily based"? At the end of the day it's just an implementation of the paper above, not sure if it is worth mentioning GPflowOpt here